Sources/CvGame.cpp

```diff
@@ -4339,7 +4309,7 @@ int CvGame::getTurnSlice() const
 
 int CvGame::getMinutesPlayed() const
 {
-	return (getTurnSlice() / gDLL->getTurnsPerMinute());
```

Comment:
I don't think this is an improvement. Abstracting the value access is more flexible, and has no overhead (the compiler will inline it). In this case it probably doesn't matter, but I would not suggest this as a general improvement to apply everywhere.